ContractExamples/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[target.wasm32-unknown-unknown]
+rustflags = ["-C", "target-feature=+bulk-memory"]
```

Comment:
Why do we need this?

Reply:
this came from Stella. I think we can remove it